docs/servo-multi-turn.md

```diff
@@ -0,0 +1,84 @@
+<!--remove-start-->
+
+# Servo - Multi-Turn
+
+<!--remove-end-->
+
+
+
+
+
+
+
+
+Run this example from the command line with:
+```bash
+node eg/servo-multi-turn.js
+```
+
+
+```javascript
+var five = require("johnny-five");
+var board = new five.Board();
+
+board.on("ready", function() {
+  var servo = new five.Servo({
+    pin: 10,
+    pwmRange: [600, 2370], // Cheapo servo has limited PWM range
+    deviceRange: [0, 2430] // This multi-turn servo can turn 2430 degrees or 6.75 revolutions
```

Comment:
nit: put comments above the line (throughout), like:

```js
// Cheapo servo has limited PWM range
pwmRange: [600, 2370],
 // This multi-turn servo can turn 2430 degrees or 6.75 revolutions
deviceRange: [0, 2430]
```

The reason I ask for this is to improve johnny-five.io readiability on non-desktop/laptop devices